Lib/bdb.py

```diff
@@ -194,6 +277,12 @@ def user_exception(self, frame, exc_info):
         pass
 
     def _set_stopinfo(self, stopframe, returnframe, stoplineno=0):
+        """Set the attributes for stopping.
+
+        If `stoplineno` is greater than or equal to 0, then stop at line
```

Comment:
Thanks.  I've removed the backticks on the ones I added.  There is still one set of backticks that I didn't add.  Do you think I should remove those too?